### PR TITLE
RATEST-138: Fix Delete Visit Type Test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID: https://issues.openmrs.org/browse/RATEST-138

Description:
Failing Test on firefox: DeleteVisitTypeTest

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ public VisitTypePage goToVisitType(String name) {
 
     public List<String> getVisitTypeList() {
         List<String> visitTypeList = new ArrayList<String>();
+        driver.navigate().refresh();
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java:48]: any reason why u want to refresh the page here?

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java:48]: The element goes Into stale condition; either the element is read s if its no longer attached to the DOM, or Its read as if it is not in the current frame context. Refreshing the page makes the element accessible

Comment (k-joseph) [reply ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java:48]: Are u only able to resolve that this way only?

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java:48]: A smart way for the resolution is in the incoming push.

File: ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java [hunk 2]
```diff
@@ -45,7 +46,7 @@ public VisitTypePage goToVisitType(String name) {
 
     public List<String> getVisitTypeList() {
         List<String> visitTypeList = new ArrayList<String>();
-        driver.navigate().refresh();
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java]: This does the required magic and sorts out the element to be retrievable

File: ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java [hunk 3]
```diff
@@ -45,7 +46,7 @@ public VisitTypePage goToVisitType(String name) {
...
+        waiter.until(ExpectedConditions.visibilityOfElementLocated(VISIT_TYPE_LIST));
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/VisitTypeListPage.java:49]: This is smart enough to make the element retrievable

**Pull request merged**